.github/workflows/docs-preview.yml

```diff
@@ -0,0 +1,16 @@
+name: Preview docs on Netlify
+
+on:
+  pull_request:
+    branches:
+      - main
+      - release-2.x
+    paths:
+      - docs/**
+
+jobs:
+  preview:
+    uses: apollographql/docs/.github/workflows/preview.yml@main
+    secrets:
+      NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
```

Comment:
It's easier to have it as a secret because all of the repos will be referencing it. So one source of truth rather than 11ish plain-text site IDs in different repos.